### PR TITLE
Issue 188: add orchestrator runtime recipe fixtures

### PR DESCRIPTION
**Pull request by pivox**
## Summary
- Add Fake/Paper dry-run dashboard fixtures for the R1-R16 orchestrator runtime recipe.
- Document prerequisites, PASS/FAIL/BLOCKED matrix, evidence, cleanup, rollback, and Temporal schedule probes.
- Add fixture validation tests covering JSON shape, dry-run invariants, repeated application without duplicates, server payload generation, and secret-like value scanning.

## Verification
- `python3 -m pytest tests/test_runtime_recipe_fixtures.py -q` from `python-orchestrator` → 5 passed.
- `python3 -m pytest` from `python-orchestrator` → 410 passed, 3 skipped.
- `python3 -m json.tool` on both runtime recipe fixtures.
- `python3 -m mkdocs build --strict`.
- `python3` YAML parse for `mkdocs.yml`.
- Secret-like value scan on runtime recipe docs/fixtures.
- `git diff --check`.

Part of #188
Related to #173

References previous roadmap/dependency work by content: #189, #190, #205, #206, #212.

### CODE REVIEW

**Comment by pivox** [comment]
@codex please review this PR. Focus on correctness against #188, fixture idempotence, docs accuracy, safety/no-live guarantees, missing tests, and whether the recipe could accidentally imply production behavior. Please post review threads on specific lines when useful, or a PR comment if feedback is general.

File: docs/handbook/runbooks/orchestrator-runtime-recipe-r1-r16.md [hunk 1]
```diff
@@ -0,0 +1,236 @@
...
+6. relire les sets et verifier l'unicite de `set_id`.
+
+Exemple d'application manuelle du dashboard nominal :
+
+```bash
+curl -sS -X POST http://localhost:8099/dashboards \
+  -H 'Content-Type: application/json' \
+  -d '{"name":"recipe-r1-r16-nominal-fake","enabled":true,"description":"Recette runtime R1-R16 - profils regular, scalper et scalper_micro en Fake/Paper dry-run."}'
+
+curl -sS -X POST http://localhost:8099/dashboards/1/sets \
```
[the remark this answers is not in the document]
Comment (pivox) [reply docs/handbook/runbooks/orchestrator-runtime-recipe-r1-r16.md]: Fixed in commit 5e7ee5d. The runbook now resolves `RECIPE_DASHBOARD_ID` by `dashboard.name`, creates the dashboard only if absent, and uses that variable for set creation, manual run, schedule creation, and cleanup instead of hard-coding dashboard id 1.

File: docs/handbook/runbooks/orchestrator-runtime-recipe-r1-r16.md [hunk 2]
```diff
@@ -0,0 +1,236 @@
...
+| R7 - Echec partiel | degraded | Mixer un set nominal et un set en erreur. | `partial_failure`, compteurs success/failed exacts. |
+| R8 - Idempotence | nominal | Rejouer le meme `idempotency_key`. | Aucun doublon `runs`/`run_sets`, replay explicite du resultat. |
+| R9 - Replay explicite | nominal | Rejouer un run termine via la meme cle ou le meme tick. | Relation lisible avec le run initial, aucun effet de bord duplique. |
+| R10 - Crash et reprise | degraded ou nominal | Interrompre l'orchestrateur apres claim, attendre expiration puis relancer. | Claim repris, set non perdu, pas de double execution observee. |
+| R11 - Chevauchement schedules | degraded, `recipe_fake_overlap_scalper` | Declencher deux ticks proches avec meme dashboard. | Lock ou `running` in-flight visible, pas de double dispatch incompatible. |
+| R12 - Contention entre profils | nominal + degraded | Cibler `BTCUSDT` depuis deux profils Fake dry-run. | Distinction lock orchestration / lock metier, aucun live. |
+| R13 - Snapshot obsolete | degraded | Fournir un snapshot absent ou incoherent via stub open-state. | Fail-closed documente, pas de fallback exchange silencieux. |
```
[the remark this answers is not in the document]
Comment (pivox) [reply docs/handbook/runbooks/orchestrator-runtime-recipe-r1-r16.md]: Fixed in commit 5e7ee5d. R13 now states that the dry-run fixtures cannot validate the live snapshot fail-closed branch: absent snapshots are non-blocking for effective dry-run sets. The row now requires `BLOCKED` until a dedicated safe local harness can exercise an effective `fake/demo` live set without mainnet or a real exchange, while keeping the final expected proof as fail-closed before dispatch.

**Comment by pivox** [comment]
@codex review
